.github/workflows/pr_checks.yml

```
@@ -585,6 +672,110 @@ jobs:
+      # Auto-assign PR based on check results
+      - name: Auto-assign PR based on results
+        if: ${{ always() && steps.check-excluded-bot.outputs.excluded_bot_detected != 'true' }}
+        uses: actions/github-script@v7.0.1
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const { owner, repo } = context.repo;
+
+            // Resolve PR number: prioritize manual trigger, fall back to PR event
+            const manualTriggerPR = '${{ needs.check-manual-trigger.outputs.pr_number }}';
+            const prNumber = (manualTriggerPR && !isNaN(Number(manualTriggerPR))) 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)